src/libmongoc/tests/unified/runner.c

```diff
@@ -617,10 +630,9 @@ test_setup_initial_data (test_t *test, bson_error_t *error)
       }
 
       /* Insert documents if specified. */
-      if (bson_has_field (&collection_data, "documents")) {
+      if (documents) {
```

Comment:
If my comment above about this being required is correct then we could remove this.

Reply:
I changed this to check whether the array is empty.